storybook/webpack.config.ts

```diff
@@ -83,7 +83,6 @@ const svgIcons: Rule = {
 
 const elm: Rule = {
   test: /\.elm$/,
-  exclude: [/elm-stuff/, /node_modules/],
   use: [
```

Comment:
Why was this rule removed?

Reply:
This is being overwritten by the condition to exclude external modules, so was only causing confusion. Must have gotten caught up in a copy-and-paste of the elm loader rule.